app/views/admin/AdministerUserModal.coffee

```diff
@@ -207,3 +212,38 @@ module.exports = class AdministerUserModal extends ModalView
         @render()
       ), 2000)
     null
+
+  onClickEditPrepaidsInfoButton: (e) ->
+    prepaidId=@$(e.target).data('prepaid-id')
+    @prepaidTableState[prepaidId] = 'editMode'
+    @renderSelectors('#'+prepaidId)
+
+  onClickCancelPrepaidInfoEditButton: (e) ->
+    @prepaidTableState[@$(e.target).data('prepaid-id')] = 'viewMode'
+    @renderSelectors('#'+@$(e.target).data('prepaid-id'))
+
+  onClickSavePrepaidInfo: (e) ->
+    prepaidId= @$(e.target).data('prepaid-id')  
+    prepaidStartDate= @$el.find('#'+'startDate-'+prepaidId).val()
+    prepaidEndDate= @$el.find('#'+'endDate-'+prepaidId).val()
+    prePaidTotalLicenses=@$el.find('#'+'totalLicenses-'+prepaidId).val()
+    @prepaids.each (prepaid) =>
+      if (prepaid.get('_id') == prepaidId) 
+        #validations
+        unless prepaidStartDate and prepaidEndDate and prePaidTotalLicenses
+          return
+        if(prepaidStartDate >= prepaidEndDate)
+          alert('End date cannot be on or before start date')
+          return
+        if(prePaidTotalLicenses < (prepaid.get('redeemers') || []).length)
+          alert('Total number of licenses cannot be less than used licenses')
+          return
+        prepaid.set('startDate', moment.timezone.tz(prepaidStartDate, "America/Los_Angeles").toISOString())
+        prepaid.set('endDate',  moment.timezone.tz(prepaidEndDate, "America/Los_Angeles").toISOString())
+        prepaid.set('maxRedeemers', prePaidTotalLicenses)
+        options = {}
+        prepaid.patch(options)
```

Comment:
Please check if it is required to break from the loop (line 230) here since the prepaid is updated

Reply:
Good catch. Added a return once the prepaid id is found and save is done.